packages/fxa-payments-server/src/routes/ProductV2/SubscriptionCreate/index.tsx

```diff
@@ -222,7 +230,19 @@ async function handleSubscriptionPayment({
   onRetry: (status: RetryStatus) => void;
   onSuccess: () => void;
 } & SubscriptionCreateAPIs) {
-  // 1. Create the payment method.
```

Comment:
I think this is in the wrong component - this is for creating a new subscription, where I think we want to use this SetupIntent stuff over in payment method updates in [the SubscriptionItem and PaymentUpdateForm neighborhood of components](https://github.com/mozilla/fxa/blob/main/packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionItem.tsx#L75)

Reply:
Thanks for the clarification. Reading the docs that were linked, it seemed like we needed to use the SetupIntents for the initial payment method creation during checkout. I'll move this work 😅